gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
 
-kotlin = "1.8.20" # should match Gradle's embedded Kotlin version https://docs.gradle.org/current/userguide/compatibility.html#kotlin
-kotlin-dokka = "1.8.20"
-kotlinx-serialization = "1.5.1"
+kotlin = "1.9.0" # should match Gradle's embedded Kotlin version https://docs.gradle.org/current/userguide/compatibility.html#kotlin
```

Comment:
[Kotlin 1.9.10](https://github.com/JetBrains/kotlin/releases/tag/v1.9.10) was released last week, but as there's a note here to match Gradle's versioning I've kept it to 1.9.0 for now